### PR TITLE
PR to add Spark 2.4.4

### PR DESCRIPTION
**Pull request by rnarasimhan88**
Add Spark 2.4.4 to sdkman and make it default spark version

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/SparkMigrations.scala [hunk 1]
```diff
@@ -55,11 +55,18 @@ class SparkMigrations {
...
+    Seq("2.4.1", "2.4.2", "2.4.3").foreach { version =>
       Version("spark", version, s"https://archive.apache.org/dist/spark/spark-$version/spark-$version-bin-hadoop2.7.tgz")
         .validate()
         .insert()
     }
-    setCandidateDefault("spark", "2.4.4")
+    setCandidateDefault("spark", "2.4.3")
+  }
+
+  @ChangeSet(order = "008", id = "008-add_spark_2.4.4", author = "marc0der")
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/SparkMigrations.scala]: `author` should be your github username.

**Comment by rnarasimhan88** [comment]
@marc0der I have added 008 as suggested

File: src/main/scala/io/sdkman/changelogs/SparkMigrations.scala [hunk 2]
```diff
@@ -55,11 +55,18 @@ class SparkMigrations {
...
+  def migration008(implicit db: MongoDatabase) = {
+    Version("spark", "2.4.4", "https://archive.apache.org/dist/spark/spark-2.4.4/spark-2.4.4-bin-hadoop2.7.tgz")
+      .validate()
+      .insert()
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/SparkMigrations.scala]: After inserting, you can also call the `.asCandidateDefault()` to make this the default version.

Comment (rnarasimhan88) [reply src/main/scala/io/sdkman/changelogs/SparkMigrations.scala]: Done.

File: src/main/scala/io/sdkman/changelogs/SparkMigrations.scala (hunk 1 above)
Comment (rnarasimhan88) [reply src/main/scala/io/sdkman/changelogs/SparkMigrations.scala]: Oops, sorry missed it while copy-paste

Comment (rnarasimhan88) [reply src/main/scala/io/sdkman/changelogs/SparkMigrations.scala]: This is fixed

**Comment by marc0der** [comment]
Thanks for your contribution!

**Comment by rnarasimhan88** [comment]
@marc0der Thank you for giving me quick feedback that helped me to make the necessary corrections. Glad that I could be of some help.

**Pull request merged**